src/meta/src/manager/diagnose.rs

```diff
@@ -626,7 +626,11 @@ impl DiagnoseCommand {
         let compute_clients = ComputeClientPool::adhoc();
         for worker_node in &worker_nodes {
             if let Ok(client) = compute_clients.get(worker_node).await
-                && let Ok(result) = client.stack_trace().await
+                && let Ok(result) = client
+                    .stack_trace(StackTraceRequest {
+                        actor_traces_format: Some("json".to_owned()),
```

Comment:
ditto, please add a parameter to the `/diagnose` API instead.

Reply:
Example:
```
curl http://localhost:5691/api/monitor/diagnose/\?actor_traces_format\=json > diagnose.txt
```